src/solver/main/economy.cpp

```diff
@@ -38,10 +39,15 @@ void Application::runSimulationInEconomicMode()
 {
     // Type of the simulation
     typedef Solver::Simulation::ISimulation<Solver::Simulation::Economy> SimulationType;
-    SimulationType simulation(*pStudy, pSettings);
+    SimulationType simulation(*pStudy, pSettings, &pTimeElapsedAggregator);
     simulation.run();
+
     if (!(pSettings.noOutput || pSettings.tsGeneratorsOnly))
+    {
+        TimeElapsed::Timer timer("Survey report", "synthesis_export", true, &pTimeElapsedAggregator);
```

Comment:
Consider replacing **"Survey report"** with **"Synthesis export"**, as everybody in the team (should) understand it.